Tasks/VsTest/task.json

```diff
@@ -47,11 +48,24 @@
     "inputs": [
         {
             "name": "testAssembly",
-            "type": "string",
+            "type": "multiLine",
             "label": "Test Assembly",
-            "defaultValue": "**\\*test*.dll;-:**\\obj\\**",
+            "defaultValue": "**\\*test*.dll\n!**\\obj\\**",
             "required": true,
-            "helpMarkDown": "Test binaries to run tests on.  Wildcards can be used.  For example, `**\\*test*.dll;-:**\\obj\\**` for all dlls with test in name while excluding files in any sub-directory named obj.",
+            "helpMarkDown": "Test binaries to run tests on. Supports multiple lines of minimatch patterns.",
```

Comment:
>helpMarkDown [](start = 13, length = 12)

Also, as the pattern is being changed and new Search criteria is being added, hence vsts-tasks/Tasks/VsTest/README.md needs to be updated